app-modules/notification/src/Filament/Actions/SubscribeBulkAction.php

```diff
@@ -36,28 +36,64 @@
 
 namespace AdvisingApp\Notification\Filament\Actions;
 
-use AdvisingApp\Notification\Actions\SubscriptionToggle;
-use AdvisingApp\Notification\Models\Contracts\Subscribable;
+use AdvisingApp\Prospect\Models\Prospect;
+use AdvisingApp\StudentDataModel\Models\Student;
+use App\Models\User;
+use Exception;
+use Filament\Forms\Components\Select;
+use Filament\Forms\Components\Toggle;
+use Filament\Notifications\Notification;
 use Filament\Tables\Actions\BulkAction;
 use Illuminate\Database\Eloquent\Collection;
 
-class SubscribeBulkAction extends BulkAction
+class SubscribeBulkAction
 {
-    protected function setUp(): void
+    public static function make(string $context): BulkAction
     {
-        parent::setUp();
+        return BulkAction::make('bulkSubscription')
+            ->icon('heroicon-s-bell')
+            ->modalHeading('Create Bulk Subscription')
+            ->modalDescription(fn (Collection $records) => "You have selected {$records->count()} {$context} to subscribe.")
```

Comment:
Would it be possible to use the string helper on `{$context}` to make is singular or plural based on the `$records->count()`?